waf-tools/python_patched.py

```diff
@@ -241,7 +242,12 @@ def check_python_headers(conf):
 	if not pybin:
 		conf.fatal('Could not find the python executable')
 
-	v = 'prefix SO LDFLAGS LIBDIR LIBPL INCLUDEPY Py_ENABLE_SHARED MACOSX_DEPLOYMENT_TARGET LDSHARED CFLAGS LDVERSION'.split()
```

Comment:
I don't remember anymore why this `python_patched` module exists.  This is based on waf's own `python` "tool".  I might have applied some patch to it, to fix what, I can't remember anymore, been so long.

Perhaps it is worth checking that the normal `python` tool in WAF works as expected without any patching?